src/app/Library/CrudPanel/Traits/Views.php

```diff
@@ -308,6 +310,9 @@ public static function getFirstFieldView($viewPath, $viewNamespace = false)
         }
 
         // if no view exists, in any of the directories above... no bueno
+        if (! backpack_pro()) {
+            throw new BackpackProRequiredException('Cannot find the field view: '.$viewPath.'. Please check for typos. Alternatively, maybe you are trying to use a <a href="https://backpackforlaravel.com/products/pro-for-unlimited-projects" target="_blank">PRO field</a>?', 1);
```

Comment:
Same thing here - cannot use links yet:

```suggestion
            throw new BackpackProRequiredException('Cannot find the field view: '.$viewPath.'. Please check for typos.' . (backpack_pro() ? '' : ' If you are trying to use a PRO field, please first purchase and install the backpack/pro addon from backpackforlaravel.com'), 1);
```